MekHQ/src/mekhq/campaign/stratcon/StratconCampaignState.java

```diff
@@ -0,0 +1,233 @@
+        this.setContract(contract);
+    }
+
+    /**
+     * The opfor BV multiplier. Intended to be additive.
+     * @return The additive opfor BV multiplier.
+     */
+    public double getGlobalOpforBVMultiplier() {
+        return globalOpforBVMultiplier;
+    }
+    
+    public StratconTrackState getTrack(int index) {
+        return tracks.get(index);
+    }
+    
+    public List<StratconTrackState> getTracks() {
+        return tracks;
+    }
+    
+    public void addTrack(StratconTrackState track) {
+        tracks.add(track);
+    }
+    
+    public int getSupportPoints() {
+        return supportPoints;
+    }
+    
+    public void addSupportPoints(int number) {
+        supportPoints += number;
+    }
+    
+    public void setSupportPoints(int supportPoints) {
+        this.supportPoints = supportPoints;
+    }
+
+    public int getVictoryPoints() {
+        return victoryPoints;
+    }
+
+    public void setVictoryPoints(int victoryPoints) {
+        this.victoryPoints = victoryPoints;
+    }
+    
+    public void updateVictoryPoints(int increment) {
+        victoryPoints += increment;
+    }
+
+    public String getBriefingText() {
+        return briefingText;
+    }
+
+    public void setBriefingText(String briefingText) {
+        this.briefingText = briefingText;
+    }
+
+    public boolean strategicObjectivesBehaveAsVPs() {
+        return strategicObjectivesBehaveAsVPs;
+    }
+
+    public void setStrategicObjectivesBehaveAsVPs(boolean strategicObjectivesBehaveAsVPs) {
+        this.strategicObjectivesBehaveAsVPs = strategicObjectivesBehaveAsVPs;
+    }
+
+    public List<String> getGlobalScenarioModifiers() {
+        return globalScenarioModifiers;
+    }
+
+    public void setGlobalScenarioModifiers(List<String> globalScenarioModifiers) {
+        this.globalScenarioModifiers = globalScenarioModifiers;
+    }
+
+    public int getPendingStrategicObjectiveCount() {
+        return pendingStrategicObjectiveCount;
+    }
+
+    public void setPendingStrategicObjectiveCount(int pendingStrategicObjectiveCount) {
+        this.pendingStrategicObjectiveCount = pendingStrategicObjectiveCount;
+    }
+    
+    public void incrementPendingStrategicObjectiveCount(int increment) {
+        this.pendingStrategicObjectiveCount += increment;
+    }
+    
+    public void incrementPendingStrategicObjectiveCount() {
+        pendingStrategicObjectiveCount++;
+    }
+    
+    public int getStrategicObjectiveCompletedCount() {
+        return completedStrategicObjectiveCount;
+    }
+    
+    public void incrementStrategicObjectiveCompletedCount() {
+        completedStrategicObjectiveCount++;
+    }
+    
+    public void decrementStrategicObjectiveCompletedCount() {
+        completedStrategicObjectiveCount--;
+    }
+    
+    public void useSupportPoint() {
+        supportPoints--;
+    }
+    
+    public void convertVictoryToSupportPoint() {
+        victoryPoints--;
+        supportPoints++;
+    }
```

Comment:
Should this ensure victoryPoints > 0?

Reply:
I'd prefer for this class to be pretty dumb and for most of the rules logic to reside in the RulesManager class.